alts/src/main/java/io/grpc/alts/internal/AltsProtocolNegotiator.java

```diff
@@ -298,6 +298,7 @@ synchronized Channel get() {
     synchronized void close() {
       if (channel != null) {
         channelPool.returnObject(channel);
+        channel = null;
```

Comment:
You can also do:
```
channel = channelPool.returnObject(channel);
```
which is the recommended idiom.

Reply:
Thanks for advice. Done.